code/game/turfs/open/openspace.dm

```diff
@@ -95,6 +95,15 @@ GLOBAL_DATUM_INIT(openspace_backdrop_one_for_all, /atom/movable/openspace_backdr
 		return TRUE
 	return FALSE
 
+/turf/open/transparent/openspace/can_traverse_safely(atom/movable/traveler)
+	var/turf/destination = GET_TURF_BELOW(src)
+	if(!destination)
+		return TRUE // this shouldn't happen
+	if(!traveler.can_zTravel(destination, DOWN, src)) // something is blocking their fall!
+		return FALSE
+	if(!traveler.can_zFall(src, DOWN, destination)) // they can't fall!
+		return FALSE
+	return TRUE
```

Comment:
doesn't this just mean they can always traverse openspace? which means mobs will like leap off ledges to get across gaps as its a cheaper path? for on z-level movements?